eq-author-api/schema/tests/validation.test.js

```diff
@@ -64,16 +64,14 @@ describe("validation", () => {
 
   describe("Page validation", () => {
     it("contains validation error info when querying page", async () => {
-      const validationErrorInfo = {
-        [`page_${page.id}`]: {
-          errors: [
-            {
-              field: "field",
-              errorCode: "ERR_CODE",
-            },
-          ],
+      const validationErrorInfo = [
+        {
+          id: page.id,
+          type: "page",
```

Comment:
Should this type be `pages` as in questionPage? Could this be a constant?

Reply:
Yup, now changed to pages and added as constant